src/workspaces/dashboard/WorkspaceNotifications.test.tsx

```diff
@@ -0,0 +1,157 @@
+} from 'src/profile/notification-settings/utils';
+import { renderWithAppContexts as render } from 'src/testing/test-utils';
+import { defaultGoogleWorkspace } from 'src/testing/workspace-fixtures';
+import { WorkspaceNotifications } from 'src/workspaces/dashboard/WorkspaceNotifications';
+
+type NotificationUtilsExports = typeof import('src/profile/notification-settings/utils');
+jest.mock('src/profile/notification-settings/utils', (): NotificationUtilsExports => {
+  const originalModule = jest.requireActual<NotificationUtilsExports>('src/profile/notification-settings/utils');
+  return {
+    ...originalModule,
+    // Mock errors so that we can test that a notification is sent
+    updateNotificationPreferences: jest.fn().mockRejectedValue(new Error('test error')),
+  };
+});
+
+describe('WorkspaceNotifications', () => {
+  const testWorkspace = {
+    ...defaultGoogleWorkspace,
+    workspace: {
+      ...defaultGoogleWorkspace.workspace,
+      name: 'name',
+      namespace: 'namespace',
+    },
+  };
+
+  beforeEach(() => {
+    // Don't show expected error responses in logs
+    jest.spyOn(console, 'error').mockImplementation(() => {});
+  });
+
+  afterEach(() => {
+    userStore.reset();
+  });
+
+  it.each([
+    {
+      profile: {
+        'notifications/SuccessfulSubmissionNotification/namespace/name': 'true',
+        'notifications/FailedSubmissionNotification/namespace/name': 'true',
+        'notifications/AbortedSubmissionNotification/namespace/name': 'true',
+      },
+      expectedState: true,
+    },
+    {
+      profile: {},
+      expectedState: true,
+    },
+    {
+      profile: {
+        'notifications/SuccessfulSubmissionNotification/namespace/name': 'false',
+        'notifications/FailedSubmissionNotification/namespace/name': 'false',
+        'notifications/AbortedSubmissionNotification/namespace/name': 'false',
+      },
+      expectedState: false,
+    },
+  ])('renders checkbox with submission notifications status', ({ profile, expectedState }) => {
+    // Arrange
+    // @ts-ignore: partial update
+    act(() => userStore.update((state) => ({ ...state, profile })));
+
+    // Act
+    const { getByLabelText } = render(<WorkspaceNotifications workspace={testWorkspace} />);
+
+    // Assert
+    const submissionNotificationsCheckbox = getByLabelText('Receive submission notifications');
+    expect(submissionNotificationsCheckbox.getAttribute('aria-checked')).toBe(`${expectedState}`);
+  });
+
+  it('calls updateNotificationPreferences when submission checkbox is clicked', async () => {
+    // Arrange
+    const user = userEvent.setup();
+
+    // Act
+    const { getByLabelText } = render(<WorkspaceNotifications workspace={testWorkspace} />);
+    const submissionNotificationsCheckbox = getByLabelText('Receive submission notifications');
+    await user.click(submissionNotificationsCheckbox);
+
+    // Assert
+    expect(updateNotificationPreferences).toHaveBeenCalledWith(
```

Comment:
The original test verified the Ajax calls, but we don't need to do that here because it is tested by the unit test of `udpateNotificationPreferences`.